workflow/exp/exp.conus3km

```diff
@@ -56,7 +56,7 @@ export COLDSTART_CYCS="00 12"   # cycles to cold start from external models
 #
 export NODES_IC="<nodes>40:ppn=20</nodes>"
 export NODES_LBC="<nodes>40:ppn=20</nodes>"
-export NODES_FCST="<nodes>40:ppn=20</nodes>"
+export NODES_FCST="<nodes>40:ppn=40</nodes>"
```

Comment:
should we apply this change to all 8 template exp setup files?

Reply:
Sorry forgot to double check this with you.

Do we need to to change `40:PPN=40` for all conus3km?
I think you missed `exp.ens_conus3km`

Reply:
Modified. Thanks @guoqing-noaa